lib/inspec/plugin/v2/activator.rb

```diff
@@ -17,5 +17,26 @@ def activated?(new_value = nil)
       return self[:'activated?'] if new_value.nil?
       self[:'activated?'] = new_value
     end
+
+    # Load a plugin, but if an error is encountered, store it and continue
+    def activate
+      return if activated?
+      # rubocop: disable Lint/RescueException
+      begin
+        impl_class = self[:activation_proc].call
+        self[:'activated?'] = true
```

Comment:
Here I guess we can define this :'activated?' as :activated? as both means the same. If you check the output of `:'activated?'` and `:activated?` on irb both are same.

Reply:
Hey, thanks for that! In Ruby 1.9, the quotes were required if the symbol name had unusual characters, but you're right, `self[:activated?]` is safe in 2.3.8, 2.4.5, and 2.5.3, so I'll us that.